src/qml/README.md

```diff
@@ -6,9 +6,17 @@ This directory contains the source code for an experimental Bitcoin Core graphic
 
 Insecure CI artifacts are available for local testing of the master branch, avoiding the need to build:
 - for Windows: [`insecure_win_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip)
-- for macOS: [`insecure_mac_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip)
+- for Intel macOS: [`insecure_mac_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip)
+- for Apple Silicon macOS: [`insecure_mac_arm64_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_arm64_gui.zip)
```

Comment:
```suggestion
- for Apple Silicon macOS: [`insecure_mac_arm64_gui.zip`](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip)
```

because of https://github.com/bitcoin-core/gui-qml/blob/3b82418c9d99ebe5198e97d071a15ab7d627c4a5/.cirrus.yml#L328

Reply:
I agree with this suggestion.

When going on the link: https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/**macos**/insecure_mac_arm64_gui.zip, I got the following message:

![Screenshot from 2022-07-05 16-32-49](https://user-images.githubusercontent.com/85434418/177313692-e8291dbf-43cc-43ba-88a9-1ff8f972e0c4.png)

However, when I visited the link in the suggestion: https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/**macos_arm64**/insecure_mac_arm64_gui.zip, I was able to download the CI artifact successfully.

Reply:
done